streams/src/test/java/org/apache/kafka/streams/state/internals/MeteredTimestampWindowStoreTest.java

```diff
@@ -89,4 +92,49 @@ public void shouldNotExceptionIfFetchReturnsNull() {
         assertNull(store.fetch("a", 0));
     }
 
+    @Test
+    public void shouldNotThrowExceptionIfSerdesCorrectlySetFromProcessorContext() {
```

Comment:
Can we add those test for `MeteredTimestampedKeyValueStoreTest`, too. We had the same bug in `MeteredTimestampedKeyValueStore` and fixed it "on the side" but never added a test...

Reply:
Will open a separate PR for that.